server/dal/stock-info.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require("mongoose");
+const dotenv = require('dotenv');
+dotenv.config();
+
+mongoose.connect(process.env.MONGO_DB_CONNECTION_STRING, { useNewUrlParser: true, useUnifiedTopology: true })
+
+var Schema = mongoose.Schema
+
+var stockInfoSchema = new Schema({
+    company_name: String,
```

Comment:
During yesterday's call we also discussed how `company_name` and `description` (which isn't listed in this schema) will be constants; `ticker` too of course. The rest of these values won't be.

Not sure if that means we should remove the `company_name` and `ticker` from this schema.

Reply:
Same as above